app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -218,6 +218,42 @@ class ProfileProgressFragmentTest {
     }
   }
 
+  @Test
+  fun testProfileProgressFragment_imageSelectAvatar_changeOrientation_checkGalleryIntent() {
+    val expectedIntent: Matcher<Intent> = allOf(
+      hasAction(Intent.ACTION_PICK),
+      hasData(MediaStore.Images.Media.EXTERNAL_CONTENT_URI)
+    )
+    val activityResult = createGalleryPickActivityResultStub()
+    intending(expectedIntent).respondWith(activityResult)
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(internalProfileId)).use {
+      waitForTheView(withText("Admin"))
+      onView(
+        atPositionOnView(
+          R.id.profile_progress_list,
+          0,
+          R.id.profile_edit_image
+        )
+      ).perform(click())
+      onView(withText(R.string.profile_progress_edit_dialog_title)).inRoot(isDialog())
+        .check(matches(isDisplayed()))
+      onView(withText(R.string.profile_picture_edit_alert_dialog_choose_from_library))
+        .perform(click())
+      intended(expectedIntent)
+      onView(isRoot()).perform(orientationLandscape())
+      intended(expectedIntent)
+      onView(
+        atPositionOnView(
+          R.id.profile_progress_list,
+          0,
+          R.id.profile_edit_image
+        )
+      ).perform(click())
+      onView(withText(R.string.profile_progress_edit_dialog_title)).inRoot(isDialog())
```

Comment:
Please add a comment to the effect: “dialog should still be open after configuration change”

Reply:
Committed directly.